### ISSUE TITLE
Install on Arch Linux

### ISSUE THREAD

**Issue by filippos13** [fusion809/MacBuntu-Remix#1]
Hello! Great work! I really loved the icons you made! How can I install it on arch linux? I am using Gnome.

**Comment by fusion809** [comment]
On any Linux distribution or Unix-like system, for that matter (so FreeBSD, NetBSD and OpenBSD are included), using GNOME, related desktops (like Cinnamon, LXDE, MATE or Xfce) or even KDE Plasma in order to install an icon theme (any theme this one included) you merely need to move the folder containing the theme's `index.theme` file to `~/.local/share/icons` (I suggest remembering this because odds are for some icon themes the maintainers won't even bother giving you this explanation I'm giving you), if you merely want to install it for your present user account. If you want to install it and make it available for all users move this folder to `/usr/share/icons` instead (which will require root privileges so you will need to do so with either `sudo` or `su`). In the case of this theme running:

```bash
git clone https://github.com/fusion809/MacBuntu-Remix ~/.local/share/icons/MacBuntu-Remix
```

should install it for your present user, assuming `git` is already installed. To install it system-wide (for all users) run:

```bash
sudo git clone https://github.com/fusion809/MacBuntu-Remix /usr/share/icons/MacBuntu-Remix
```

**Comment by fusion809** [comment]
Closed via c52e30b.

**Comment by filippos13** [comment]
Thank you! I am new to Linux and currently I am trying to learn the basics!
